modules/@apostrophecms/i18n/i18n/sk.json

```diff
@@ -183,7 +183,7 @@
   "moreOptions": "Viac možností",
   "moreOperations": "Viac úprav",
   "multipleEditors": "Viaceré editory",
-  "newDocType": "Nový {{ type }}",
+  "newDocType": "Nový(á) {{ type }}",
```

Comment:
Another option for this could be a translation of `Create {{ type }}` instead. Is this dual gender format common enough or would it be better to change the phrase a bit?

Reply:
Great point. I went with Create and had to change inflection in Tag, User and File.

Fix: b7096deaac715c5a270a69f6455f64e12bee0678